### ISSUE TITLE
[enhancement] Tool to import torrent data from external sources

### ISSUE THREAD

**Issue by jbruchon** [boramalper/magnetico#197]
It would be helpful to have a tool that could add/update torrent data to the database from external sources such as a list of magnet links, infohashes, .torrent files, or a copy of someone else's magnetico DB. This would allow external tools that can assemble such resources (e.g. a web crawler, a DB dump from a torrent hosting site, or a pile of .torrent files a user has collected) to be used to collect torrent data.

**Comment by boramalper** [comment]
I agree. I think having a clearly defined export format would be great too. I'll work on it. =)

**Comment by boramalper** [comment]
Also #187.

**Comment by jbruchon** [comment]
I'm primarily (almost exclusively) interested in importing data, not necessarily exporting it (a CSV dumped from the DB would probably be fine, really). I have access to multiple internet connections and I can siphon data from those separately, but I can't copy the DB from one and inject its data into the other which is the main thing I'd like to do. I can probably do it with SQLite tools but it'd be better to have something that can merge the data in a more automated way.

**Comment by skobkin** [comment]
I think that torrent import is pretty easy to implement with current persistence layer. Not sure about export but probably it's easy too even with current methods (like search with pagination).

**Comment by ProphetZarquon** [comment]
At the risk of repeating myself to the point of annoyance, each magneticod instance having the option of distributing its DB across each other instance via a secondary DHT seems like a one-stop-shop solution to pooling the efforts of multiple instances...

**Comment by skobkin** [comment]
I think that it would be good to use the JSON Lines format which is already supported by magneticod with one exception: we should also have `discoveredOn` field there.
